.github/workflows/prb.yml

```diff
@@ -0,0 +1,86 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Pull Request Builder
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-jdk8:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK8
+      uses: actions/setup-java@v1.4.3
+      with:
+        java-version: 1.8
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Cache Gradle packages
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.gradle/caches
+          ~/.gradle/wrapper
+        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+        restore-keys: ${{ runner.os }}-gradle
+    - name: Build with Gradle
+      run: ./gradlew --parallel --max-workers=4 clean test
+    - name: Publish Test Results
+      if: ${{ always() }}
+      uses: scacap/action-surefire-report@v1.0.5
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        report_paths: '**/build/test-results/test/TEST-*.xml'
+        check_name: JDK8 Test Report
+
+  build-jdk11:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK11
+      uses: actions/setup-java@v1.4.3
+      with:
+        java-version: 1.11
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Cache Gradle packages
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.gradle/caches
+          ~/.gradle/wrapper
+        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+        restore-keys: ${{ runner.os }}-gradle
+    - name: Build with Gradle
+      run: ./gradlew --parallel --max-workers=4 clean test
+    - name: Publish Test Results
+      if: ${{ always() }}
+      uses: scacap/action-surefire-report@v1.0.5
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        report_paths: '**/build/test-results/test/TEST-*.xml'
+        check_name: JDK11 Test Report
+
+  quality-jdk11:
```

Comment:
Because we release using JDK8 we should run `quality` task on JDK8 as well. Otherwise, there is a risk to fail the release. We observed behavior different for spotbugs on different JDK versions.